playbooks/roles/ocp-odf-operator/files/ImageContentSourcePolicy.yml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: brew-registry
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - brew.registry.redhat.io
+    source: registry.redhat.io
+  - mirrors:
+    - brew.registry.redhat.io
+    source: registry.stage.redhat.io
+  - mirrors:
+    - brew.registry.redhat.io
+    source: registry-proxy.engineering.redhat.com
```

Comment:
Can we re-use this role [set-custom-catalogsource](https://github.com/ocp-power-automation/ocp4-playbooks-extras/blob/main/playbooks/roles/set-custom-catalogsource/tasks/main.yml) for `CatalogSource` and `ImageContentSourcePolicy` ?

Reply:
Done